Python/libraries/recognizers-choice/recognizers_choice/resources/english_choice.py

```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------------------
+# <auto-generated>

```

Comment:
Why are there two versions of the auto-gen resources code?
Python/libraries/recognizers-choice/recognizers_choice/resources/english_choice.py
and
Python/libraries/recognizers-choice/resources/english_choice.py
The second one seems to be the correct one?

Reply:
Indeed, I am deleting this file. Thanks!